src/viper/trans.ml

```diff
@@ -109,6 +106,7 @@ type ctxt =
     ids : (sort * T.t) T.Env.t;
     label_to_tmp_var : id String_map.t; (* Motoko label -> associated tmp variable *)
     label_to_vpr_label : string String_map.t; (* Motoko label -> Viper label *)
+    reqs : reqs;
```

Comment:
Could you add some comment here? I guess "reqs" stands for "requirements" but it's too generic and for someone who unfamiliar with the code could be very confused.

Reply:
Added a comment